OutOfSchool/OutOfSchool.WebApi/Services/StatisticService.cs

```diff
@@ -168,7 +168,7 @@ public async Task<IEnumerable<WorkshopCard>> GetPopularWorkshopsFromDatabase(int
         if (catottgId > 0)
         {
             workshops = workshops
-                .Where(w => w.Address.CATOTTGId == catottgId);
+                .Where(w => w.Address.CATOTTGId == catottgId || (w.Address.CATOTTG.Category == "B" && w.Address.CATOTTG.ParentId == catottgId));
```

Comment:
Please use the `CodeficatorCategory.CityDistrict.Name` "constant" instead of hardcoding "B"

Reply:
Thank you for your comment. I changed it.